dev-python/cloudscraper/cloudscraper-1.2.69-r1.ebuild

```diff
@@ -1,10 +1,10 @@
-# Copyright 1999-2023 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 DISTUTILS_USE_PEP517=setuptools
-PYTHON_COMPAT=( python3_{9..11} )
```

Comment:
Does 13 work too for hydrus+deps? (default change is soon)

Reply:
I'll check.

Reply:
Unfortunately at least js2py which cloudscraper requires (optionally, to run tests, at least) is broken on 3.13 and while there exists a patch in a separate repository by someone else that fixes it (https://github.com/a-j-albert/Js2Py---supports-python-3.13), I'm not sure if I should be the one to change the package, since it's assigned to someone else.

Reply:
Please do in a followup if it isn't done by now.